package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "2020-04-22-noisemaker",
+  "version": "1.0.0",
+  "description": "## Prize",
```

Comment:
Once deployed, it's a good idea to update these fields to be personalized to your project.